src/SDKs/NetApp/Management.NetApp/Microsoft.Azure.Management.NetApp.csproj

```diff
@@ -8,11 +8,17 @@
     <Description>Provides NetApp storage management capabilities for Microsoft Azure.</Description>
     <AssemblyTitle>Microsoft Azure NetApp Management</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Management.NetApp</AssemblyName>
-    <Version>0.9.0-preview</Version>
+    <Version>0.9.1-preview</Version>
     <PackageTags>MicrosoftAzure Management;NetApp</PackageTags>
     <PackageReleaseNotes>
       <![CDATA[
-        This is a public preview release of the NetApp storage management SDK. It provides the capability to create multiple file systems volumes through Microsoft Azure.
+        Version 0.9.1-preview relates to NetApp Files (ANF) RP R3
+           - Addition of export policy for volumes
+           - Addition of active directory for accounts
+        Version 0.9.0-preview relates to NetApp Files (ANF) RP R2
```

Comment:
nit: You don't need to explain what previous versions of the package do.,

Reply:
Ok. I'll keep this in mind but would like to leave it here this time.